packages/react/src/patterns/sub-patterns/Card/Card.js

```diff
@@ -74,33 +82,42 @@ function optionalContent(copy) {
  *
  * @param {object} cta cta object
  * @param {string} type type of card (clickable/static)
- * @param {string} href url for card
- * @param {object} Icon cta icon for card
  * @returns {object} JSX object
  */
-function renderFooter(cta, type, href, Icon) {
+function renderFooter(cta, type) {
   return (
     <footer className={`${prefix}--card__footer`}>
-      {type !== 'link' ? (
-        <CTA style="text" type={cta.type} href={href} copy={cta.copy} />
-      ) : (
-        Icon && <Icon />
-      )}
+      {cta &&
+        (type !== 'link' ? (
+          <CTA
+            style="text"
+            type={cta.type}
+            href={cta.href}
+            copy={cta.copy}
+            customClassName={`${prefix}--card__cta`}
+          />
+        ) : (
+          cta.icon.src && (
+            <cta.icon.src className={`${prefix}--card__cta`} {...cta.icon} />
+          )
+        ))}
     </footer>
   );
 }
 
 Card.propTypes = {
-  title: PropTypes.string.isRequired,
-  icon: PropTypes.object,
+  heading: PropTypes.string,
+  eyebrow: PropTypes.string,
   copy: PropTypes.string,
-  href: PropTypes.string.isRequired,
   cta: PropTypes.shape({
     type: PropTypes.string,
     copy: PropTypes.string,
+    href: PropTypes.string,
+    icon: PropTypes.object,
   }),
```

Comment:
Sorry if I've already commented on this (I'm losing track of what i've said where 😱) and we decided otherwise, but could we use `instanceOf` here?

Reply:
yep!